addons/website/views/snippets/snippets.xml

```diff
@@ -1080,16 +1080,16 @@
         <we-row class="o_we_theme_colors_selector">
             <we-colorpicker data-name="color_1_opt" title="Primary"
                             data-customize-website-color="" data-color="o-color-1"
-                            data-use-css-color="true"/>
+                            data-use-css-color="true" data-selected-tab="1"/>
```

Comment:
Did not you give names to the tabs in the `this.tabs` array? They should be selected by name instead then.